src/fragments/lib/logging/common/setup_logging/setup_logging.mdx

```diff
@@ -0,0 +1,95 @@
+The Amplify Logger enables you to troubleshoot and debug issues with your apps, to help you provide the best experience for your customers. You can log messages for errors by the Amplify library and add custom logs as well and send them to Amazon CloudWatch. With the Amplify Logger, you also can remotely change your logging configuration to adjust your logging levels, or add an allow list of customer IDs to help you detect issues more granularly for your apps in production.
+## Prerequisites
+
+import iosPreReq from '/src/fragments/lib/logging/ios/setup_logging/10_pre_req.mdx';
+import androidPreReq from '/src/fragments/lib/logging/android/setup_logging/10_pre_req.mdx';
+
+<Fragments
+  fragments={{
+    ios: iosPreReq,
+    android: androidPreReq
+  }}
+/>
+
+## Provision backend resources
+
+<Callout>
+
+You can use Amplify CLI generate a CDK template and [add custom resources](https://docs.amplify.aws/cli/custom/cdk/).  
```

Comment:
You can use the Amplify CLI to [add custom CDK resources](https://docs.amplify.aws/cli/custom/cdk/).

Reply:
Text updated.